examples/website-demos/social-messaging/src/components/CooldownTimer/SocialCooldownTimer.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react';
```

Comment:
did you need to create your own component here vs using the one from the library? i ask because we used the library version on the virtual event app